### PR TITLE
Use `::set-output` to print the master and PR versions

### PR DESCRIPTION
**Pull request by DilumAluthge**
@SaschaMann Can you try this branch and see if it meets your needs for #16?

### CODE REVIEW

**Comment by DilumAluthge** [comment]
Bump @SaschaMann Have you had a chance to try this out?

**Comment by SaschaMann** [comment]
@DilumAluthge Not yet, sorry. I'll get to it today or tomorrow.

**Comment by SaschaMann** [comment]
Okay, so it does what it needs to do. But I thought about how I could integrate this into the workflow and the script that detects what kind of change it is seems rather complicated (to me at least). Perhaps it would be best to add another output that returns which part of the version changed, e.g. `major`, `minor`, `patch` or `dev`. I could add a PR for this if you don't have the time. Having the version numbers available is helpful on its own as well.

Test run: https://github.com/julia-actions/Example.jl/pull/5/checks?check_run_id=348677081

File: src/main.jl [hunk 1]
```diff
@@ -3,11 +3,17 @@ import Pkg
...
+              set_output::Bool = true,
+              set_output_io::IO = stdout)::Nothing
+    head_version = get_head_version(repo_url; build_directory = build_directory)  
     master_version = get_upstream_version(repo_url; master_branch = master_branch)
     @info("Master version: $(master_version)")
+    @info("Head version: $(head_version)")
+    if set_output
+        println(set_output_io, "::set-output name=master_version::v$(master_version)")
+        println(set_output_io, "::set-output name=pr_version::v$(head_version)")
+    end
```
Comment (SaschaMann) [inline src/main.jl:16]: Perhaps this should be wrapped in an `if ENV["GITHUB_ACTIONS"]` block because it's useless on Travis or other CI providers?

Comment (DilumAluthge) [reply src/main.jl]: That makes sense to me! Can you add that change?

**Comment by DilumAluthge** [comment]
> Perhaps it would be best to add another output that returns which part of the version changed, e.g. `major`, `minor`, `patch` or `dev`. I could add a PR for this if you don't have the time.

That seems like a good idea. It would be great if you can add this! You can add it to this PR if you want, or you can make a separate PR.

File: src/main.jl (hunk 1 above)
Comment (DilumAluthge) [reply src/main.jl:16]: I added this. The default value for the keyword argument is now `set_output::Bool = get(ENV, "GITHUB_ACTIONS", "") == "true"`

**Comment by SaschaMann** [comment]
I'll make a new PR against this branch. Should be the easiest way I think.

**Pull request closed without merging**